listenstore/listenstore/utils.py

```diff
@@ -3,23 +3,32 @@
 import argparse
 from setproctitle import setproctitle
 import json
-
-
-## TODO read config from whereever
-def config():
-    return {
-        "kafka_server": "localhost:9092",
-        "cassandra_server": "localhost",
-        "cassandra_keyspace": "listenbrainz"
-    }
+import ConfigParser
+import os
+
+def config(opt_vars):
+    """ Parse config file else return default values """
```

Comment:
In other projects we just tend to use Python files for storing configuration. Do we actually need this? @alastair?